### ISSUE TITLE
Agenda for Apr 27, 2023

### ISSUE THREAD

**Issue by nairnandu** [web-platform-tests/interop#323]
Here is the proposed agenda for Apr 27 meeting:

- Test change proposals that need decision/review
  - #313 
  - #317 
  - #321 
- #277 
- #275 
- #289

**Comment by nairnandu** [comment]
Attendees: @bkardell, @nairnandu, @gsnedders, @nt1m, @foolip, @vmpstr, @meyerweb, @chrishtr, @dandclark, @jgraham

Brief notes:

*   Test change proposals that need decision/review
    *   [Remove /html/semantics/forms/the-input-element/selection-pointer.html from Interop 2023 #313](https://github.com/web-platform-tests/interop/issues/313)
        *   Consensus to support this change, provided the test can be updated to support selections and ranges
        - [ ] AI: @mfreed7 will review and update the issue 
    *   [Proposal to remove &lt;audio> element from contain-inline-size-replaced.html #317](https://github.com/web-platform-tests/interop/issues/317)
        *   vmpstr: Don't have specification for intrinsic size for audio element
        *   nt1m: trust Igalia’s judgement on this. 
        - [ ] AI: @bkardell will follow-up with @cathiechen and update the issue for WebKit
    *   [Change motion path &lt;ray> &lt;contain> test references as per spec change #321](https://github.com/web-platform-tests/interop/issues/321)
        *   foolip: @danielsakhapov found some spec issues and worked with @tabatkins to fix those. These are the subsequent test changes.
        *   nt1m: did ask internally. Haven’t looked in detail, but looks reasonable at first sight.
        - [ ] AI: @nt1m will follow-up internally and comment on the issue for WebKit position
*   [Investigation Effort goals and tracking #277](https://github.com/web-platform-tests/interop/issues/277)
    *   Mobile testing
        *   jgraham: Scoping out requirements for infra (what each browser needs). Putting together a doc. The next stage is to check with CI system providers on what we can and can’t do. Feels like we are making good progress
    *   Accessibility: 
        *   gsnedders: Seeing a good amount of progress
        *   jgraham: tests should be passing in Gecko
        *   chrishtr: Few hundred tests. Scoring criteria defined. On track for 100% by end of the year. Really happy with the progress
*   [Finish the Interop Team Charter #275](https://github.com/web-platform-tests/interop/issues/275)
    *   foolip: PR ready for review - https://github.com/web-platform-tests/interop/pull/102. gsnedders and jgraham have reviewed it. Need other teams to review as well. 
     - [ ] AI: @nairnandu will ping @boazsender from Bocoup to review and provide feedback.
*   [Having a Logo of Interop #289](https://github.com/web-platform-tests/interop/issues/289)
    *   Consensus: great idea! Would be a nice-to-have for 2024, if we find a designer, but not a blocker
*   chrishtr: Overall Interop score doing well. Kudos to the team.
*   meyerweb: CSS pseudo classes - should we be concerned about the #s in stable? Is the Interop score that low?
    *   gsnedders: https://wpt.fyi/results/css/selectors?label=master&label=stable&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-pseudos
    *   foolip: issue filed to have the label updated - https://github.com/web-platform-tests/wpt.fyi/issues/3286 
    *   chrishtr: dir() pseudo class is the only one that Chromium doesn’t support. Blocked on spec. Is there any way to accelerate that? What help is needed?
    *   bkardell: that is work in progress - https://github.com/whatwg/html/pull/9166#issuecomment-1515502609.
        End goal is that we should have a test for each leaf of this [chart](https://mermaid.ink/svg/pako:eNqdVlFv4jgQ_itW-nCtFFApFEqk3ioUkHbVU6S2-3LHPZjEaawmNrJNKUv47zfjGEhK95ZtH1o7_mbmm5lvXG-8WCbMC7w0l6s4o8qQp_FMEPi524wl04QbktFXRihJuCLUGMXnS8O2pNX6s4RPt7lRJRn_89UQrgls_nXme4AyeUkmDgCbIwBdGlmS6earjTbDYyoIF4ulISxnBROGrDKpGTHrBTtwQH-INuzN-EQzquLMh13OFpkUzCdLlftEKsIKynMLjR7sH4hkMrZ3Dn6o9UIVo7uvX7Z1nkKiIy5eac4TLEVJoopwu90GZIWNLLZijlxvgUwZhef74lzUcTqXxrCkjEYOgJzANYsNlwICmTWRKYQAelrzZ8ESgjYNJ5TEGc8TBFIioJlVSTLq0tIZTeSKKClNvQLUenpXXgt6b1VGd_9LD70hRcT-oQk42ruFbRPffkfdcqjFtr40pGF-ztwKEjkuoFvikEIZjS1PkTGFfFIlC0uswjUji7XJuHgGY83KaHLUoOr31ILXMAUl-bYbB03aIIJlVWdtcGRW3GRkzmu52kOcJxobpirdhvcooYcvZNv0D9IqSRie3zND5uTWllkxvcwNljeX8gWZUlcdLGoshYGUIEdwuAPguuosMmnRV9A8necwL7jCha02oLRREgwO9FzSYWj5zJHOCMU9r7Rtz0auZc00P07xA6GtMh5nrnHHGoJbAYPuhAbbnVJOi3tfj3haSOg1hhwftd5FRA2mcimSMpycP32o_HdXCG41LaDgIJPGaTUHxxcXxuBYooSl3E63oaYSFhUJKn0_DPOE77xd4G1Tcf22E2hJHg61u6gfCrhaHzfuwnPKZcUCMgAZoG5qlUvqFz7I0k2Yk8Dj3t_TCYP2eKD2_Sfjpc0aVAmqS3meB2fpMPVRmi8sOOt2u27dWvHEZEFv8bZLujIbV1b6ZT2HlH5heDD7_jmzKPyk3eRzduFvp9esjgv7LPPk5JgPn7AJ7042ahKMXII5L06vytPJNo1QjuNZbzqdji5_x3LkLMO7SSec_tLS872CKXhsJPCi2qCnmQdjV7CZF8ASppzCrT7zZmILUHz2PK5F7AVGQYu95SKB6R9z-qxo4QUphf9NvscSbqT6q3ql2cea7y2o8IKN9-YFreurTrvf6V9eX3YGnZvhsOd7a_jcH3Tbg-GwM-gPe1dX_Zut7_2QErx221edfvf6Ztjt9Xvdy_6gZ939bQ-Rx_Y_cvlaKQ). 

    *   chrishtr: Should these tests end up in Interop 2023?
    *   bkardell: Probably. Most of them pass, but some of them are broken in weird ways between browsers
    *   gsnedders: tests are within the scope of the original proposal. However, adding large # of new tests towards the end is problematic.
    *   jgraham: we should follow the process (test-change-proposal) for test changes
    *   bkardell: There was a PR that added a chunk of new tests. It grew in size as more got added and is right now at 40 individual tests. We are likely not going to need to remove tests. There are close to 20 tests that are required. Need some more time to streamline
    *   foolip can help review once ready - https://github.com/whatwg/html/pull/9166
